libs/designsystem/src/lib/components/tabs/tabs.component.spec.ts

```diff
@@ -96,4 +96,49 @@ describe('TabsComponent', () => {
       });
     });
   });
+
+  describe('Tab-bar height', () => {
+    afterAll(() => {
+      TestHelper.resetTestWindow();
+    });
+
+    describe('on screensize phone', () => {
+      beforeEach(async () => {
+        await TestHelper.resizeTestWindow(TestHelper.screensize.phone);
+      });
```

Comment:
If there's only 1 expectation inside the `describe` block, consider inlining the window resizing inside the `it` block